package.json

```diff
@@ -2341,5 +2345,11 @@
     },
     "optionalDependencies": {
         "canvas": "^2.7.0"
+    },
+    "resolutions": {
```

Comment:
Use of `resolutions` npm package to pin child dependencies
(in npm 8 we can do this natively without any other npm packages)